controllers/activemqartemis_controller_test.go

```diff
@@ -228,6 +229,148 @@ var _ = Describe("artemis controller", func() {
 		})
 	})
 
+	Context("Console config test", func() {
+		It("Exposing secured console", Label("console-expose-ssl"), func() {
+			//we need to use existing cluster to differentiate testing
+			//between openshift and k8s, also need it to check pod status
+			if os.Getenv("USE_EXISTING_CLUSTER") == "true" {
+
+				crd := generateArtemisSpec(defaultNamespace)
+				crd.Spec.DeploymentPlan.Size = 1
+
+				crd.Spec.DeploymentPlan.ReadinessProbe = &corev1.Probe{
+					InitialDelaySeconds: 1,
+					PeriodSeconds:       1,
+					TimeoutSeconds:      5,
+				}
+
+				crd.Spec.Console.Expose = true
+				crd.Spec.Console.SSLEnabled = true
+
+				isOpenshift, err := environments.DetectOpenshift()
+				Expect(err).To(BeNil())
+
+				var consoleSecret corev1.Secret
+
+				if isOpenshift {
+
+					By("deploying console secrets")
+					certData := make(map[string][]byte)
+					stringData := make(map[string]string)
+
+					brokerKs, ferr := os.ReadFile("../test/resources/broker.ks")
+					Expect(ferr).To(BeNil())
+					clientTs, ferr := os.ReadFile("../test/resources/client.ts")
+					Expect(ferr).To(BeNil())
+
+					certData["broker.ks"] = brokerKs
+					certData["client.ts"] = clientTs
+					stringData["keyStorePassword"] = "password"
+					stringData["trustStorePassword"] = "password"
+
```

Comment:
By("deploying well known secret name that the operator will look for")

is it correct that that secret is only referenced by the operator?

Reply:
I don't see it in the CR that is.

Reply:
The operator uses the secret to mount the certs into pod for broker to set up the secured console

Reply:
The console has a option for specifying the secret name:
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/api/v1beta1/activemqartemis_types.go#L347
If not set operator looks for the default name which is <cr-name>-console-secret